organizing_hub/views.py

```diff
@@ -32,9 +32,36 @@
 bsd_api = BSD().api
 
 BSD_BASE_URL = settings.BSD_BASE_URL
+
+event_promote_admin_message_template = Template("""Hi --
```

Comment:
What's reason for these being lowercase? Is it because we're using `Template` here?

Reply:
Just the lack of a clear standard? I think we just need some clear standards here. Usually I use lowercase by default for most things.